docs/CODEOWNERS

```diff
@@ -1,21 +1,21 @@
-# Subsystems have specific owners
-
-
-CMake/                                      @robertmaynard
-data/                                       @dpugmire
-docs/                                       @robertmaynard @kmorel
-examples/                                   @kmorel
-Utilities/                                  @kmorel
-viskores/                                       @robertmaynard @kmorel
+.github/                                        @vicentebolea
+.gitlab-ci.yml                                  @vicentebolea
+.gitlab/                                        @vicentebolea
+CMake/                                          @vicentebolea
+data/                                           @dpugmire
+docs/                                           @robertmaynard @kmorel
```

Comment:
Rob no longer works on the project. Take him off. You can put yourself on, or just leave me on it alone.